synapse/models/geospace.py

```diff
@@ -142,7 +142,7 @@ def _normPyTuple(self, valu):
     def indx(self, valu):
         return self.modl.type('geo:latitude').indx(valu[0]) + self.modl.type('geo:longitude').indx(valu[1])
 
-    def repr(self, norm, defval=None):
+    def repr(self, norm):
```

Comment:
the Longitude and Latitude types should also have repr implemented

Reply:
Since their system mode types are strings, they should be fine with the default impl